npm/ng-packs/packages/theme-shared/src/lib/components/internet-connection-status/internet-connection-status.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+import { InternetConnectionService } from '../../../../../core/src/lib/services/internet-connection-service';
+
+
+
+@Component({
+  selector: 'abp-internet-status',
+  styleUrls: ['./internet-connection-status.scss'],

```

Comment:
we don't use  css files in components unless it required. Because we have seperated theme project. like Lepton X, Theme basic so on.